### PR TITLE
add option for ignore system py modules while pickling

### PR DESCRIPTION
**Pull request by dmi-feo**
resolves https://github.com/ytsaurus/ytsaurus/issues/559

### CODE REVIEW

**Comment by Kontakter** [comment]
I have an embarrassing question: could you add a test for the new options?

File: yt/python/yt/wrapper/py_wrapper.py [hunk 1]
```diff
@@ -315,9 +320,27 @@ def add_file_to_compress(file, module_name, name):
 
         files_to_compress[relpath] = file
 
+    def is_system_module(_module):
+        def path_is_for_system_module(_path):
+            return any(pattern.search(_path) for pattern in system_module_patterns)
+
+        if getattr(_module, "__file__", None):
+            # regular module
```
Comment (gritukan) [inline yt/python/yt/wrapper/py_wrapper.py:328]: nip: Typically we start comments from capital letter and end with the dot

File: yt/python/yt/wrapper/py_wrapper.py [hunk 2]
```diff
@@ -315,9 +320,27 @@ def add_file_to_compress(file, module_name, name):
 
         files_to_compress[relpath] = file
 
+    def is_system_module(_module):
+        def path_is_for_system_module(_path):
```
Comment (gritukan) [inline yt/python/yt/wrapper/py_wrapper.py:324]: is_system_module_path?

**Review by Kontakter — APPROVED** [review]

**Comment by Kontakter** [comment]
There are some cosmetic errors:
```
yt/python/yt/wrapper/tests/test_operations.py:1525: [F401] 'fakemodule' imported but unused
yt/python/yt/wrapper/tests/test_operations.py:1550: [E303] too many blank lines (2)
```

**Review by Kontakter — CHANGES_REQUESTED** [review]
See comment above

**Comment by Kontakter** [comment]
Also there is a following error while running tests with system python:
```
...
yt/wrapper/py_wrapper.py", line 297, in create_modules_archive_default
    for pattern in get_config(client)["pickling"]["system_module_patterns"]
TypeError: 'NoneType' object is not iterable
```

**Comment by dmi-feo** [comment]
> Also there is a following error while running tests with system python:
> 
> ```
> ...
> yt/wrapper/py_wrapper.py", line 297, in create_modules_archive_default
>     for pattern in get_config(client)["pickling"]["system_module_patterns"]
> TypeError: 'NoneType' object is not iterable
> ```

@Kontakter  Could you tell me more about the environment you launched the test in? I didn't manage to reproduce the error. Does `test_create_modules_archive` pass in that case?

**Review by Kontakter — APPROVED** [review]
To run tests

**Pull request closed without merging**